File: java/product.java

public class Product {
	private String code;
	private String name;
	private Double price;
		
	public Product(String code, String name, Double price) {
		this.code = code;
		this.name = name;
		this.price = price;
	}
	
	public String getCode() {
		return code;
	}
	public void setCode(String code) {
		this.code = code;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public Double getPrice() {
		return price;
	}
	public void setPrice(Double price) {
		this.price = price;
	}
	
}

File: c++/product.cpp
#include <iostream>
#include <string>
#include "product.h"
using std::cout, std::cin, std::endl;

//Implementação
product::product(string newCode, string newName, double newPrice){ 
  setCode(newCode);
  setName(newName);
  setPrice(newPrice);
}

void product::setCode(string newCode){ 
    code = newCode;      
}

void product::setName(string newName){ 
    name = newName;      
}

void product::setPrice(double newPrice){ 
    price = newPrice;      
}

string product::getCode(){
	return code;
}

string product::getName(){
	return name;
}

double product::getPrice(){
	return price;
}




